src/test/resources/com/puppycrawl/tools/checkstyle/checks/orderedproperties/InputOrderedProperties.txt

```diff
@@ -0,0 +1,3 @@
+# txt files will not processed. This content is not relevant.
+1=val
+1=foo
```

Comment:
This is not a good test case since this is still a valid properties file on the inside.
Either make this an invalid properties file or make it out of order so if it was processed there would be a violation.

Reply:
This file is used for testShouldNotProcessFilesWithWrongFileExtension(). The content of this file is not relevant. I could remove line 2 and 3?

Reply:
The contents do matter. Imagine file extension check failed and this file was loaded as a property file. It would be a valid file, be loaded, and processed, and not produce any violations. Your check doesn't print violations on duplicate keys, just unsorted. This is why I said this is not a good input file.
Removing all but line still produces the same result. This is why I said to either make thsi an invalid properties file so it can't be loaded, or make it out of order to produce a violation if the file extension check fails.

Reply:
The file is unsorted, to product an possible violation.

Reply:
Done.